mimic/test/helpers.py

```diff
@@ -244,3 +244,14 @@ def validate_link_json(testCase, json_containing_links):
                                   '"href" is not a string: {0}'.format(link))
         testCase.assertIsInstance(link['rel'], text_type,
                                   '"rel" is not a string: {0}'.format(link))
+
+
+def get_template_id(case, eeapi):
+    """
+    Retrieve the template id.
```

Comment:
Needs more explanation - "retrieve the template ID from the given `EndpointApiTemplate`, failing the given test case if it's `None`.